### PR TITLE
Updating Material3 to RC01

### PR DESCRIPTION
**Pull request by florina-muntenescu**

### CODE REVIEW

**Review by bentrengrove — CHANGES_REQUESTED** [review]
Dependencies are centrally managed and then copied into each sample. You should upgrade it in the global versions file in `scripts/` or else it will be overriden the next time the samples are upgraded.

File: JetNews/gradle/libs.versions.toml [hunk 1]
```diff
@@ -28,9 +28,9 @@ compileSdk = "33"
 compose = "1.3.0-rc01"
 compose-compiler = "1.3.2"
 # @pin to be updated manually - check JetChat tests
-compose-material3 = "1.0.0-beta03"
+compose-material3 = "1.0.0-rc01"
 # @pin to be updated manually - check JetChat tests
```
Comment (astamato) [inline JetNews/gradle/libs.versions.toml:32]: should we remove the disclaimer?

Comment (bentrengrove) [reply JetNews/gradle/libs.versions.toml:32]: I'm not sure who added it but I would think so

**Review by bentrengrove — APPROVED** [review]
Thanks for fixing that up

**Pull request merged**